pysat/tests/test_files.py

```diff
@@ -134,10 +130,7 @@ def setup(self):
     def teardown(self):
         """Runs after every method to clean up previous testing."""
         remove_files(self.testInst)
-        try:
-            pysat.utils.set_data_dir(self.data_path, store=False)
-        except:
-            pass
+        pysat.utils.set_data_dir(self.data_path, store=False)
```

Comment:
except: pass not being run.  If `set_data_dir` is broken, the code has serious problems.